src/commands/LoadController.php

```diff
@@ -184,12 +184,30 @@ protected function process($data)
             }
         }
 
-        foreach (ArrayHelper::getValue($data, 'assign', []) as $userid => $permissionOrRoles) {
+        $assignData = ArrayHelper::getValue($data, 'assign', []);
+        $findBy = ArrayHelper::remove($assignData, 'findBy', 'findIdentity');
+
+        // Check function exist before execute
+        if(!method_exists(Yii::$app->user->identityClass, $findBy)) {
+            Yii::error(sprintf("Method %s doesn't exist", $findBy));
+            exit();
+        }
+
+        foreach ($assignData as $userid => $permissionOrRoles) {
+            Yii::info(sprintf("User %s start:", $userid));
+            $userInfo = call_user_func_array([ Yii::$app->user->identityClass, $findBy ], [ $userid ]);
+
+            if(is_null($userInfo) || is_null($userid = ArrayHelper::getValue($userInfo, 'id'))) {
+                Yii::warning(sprintf("User does not exist"));
+                continue;
+            }
+
             foreach ($permissionOrRoles as $permissionOrRole) {
                 try {
                     $this->actionAdd($userid, $permissionOrRole);
-                } catch (Exception $exception) {
-                    Yii::error(sprintf("%s", $exception->getMessage()));
+                } catch (yii\console\Exception $exception) {  // Role does not exist -> skip this role
```

Comment:
Change the bahvior of `actionAdd` to log an error saying the role or permission doesn't exist and return `ExitCode::UNSPECIFIED_ERROR` so we can check that return to say the role doesn't exist and continue.